src/components/style/interactive/muiButtonShim.css

```diff
@@ -0,0 +1,33 @@
+.fdsButton {
+  background: transparent;
+  border-width: 0;
+  padding: calc(var(--space-half) / 2) var(--space-half) !important;
+  border-radius: var(--border-radius-default);
+  cursor: pointer;
+  color: var(--font-color-secondary);
+  fill: var(--font-color-secondary);
+  font-weight: var(--font-weight-bold);
+  text-transform: uppercase;
+  margin: 0;
+}
```

Comment:
Is the `fdsButton` name going to conflict with our upcoming button component stuff? I wonder if we should start namespacing these different buttons now to avoid naming clashes & semantic confusion (so `fdsMuiButton`, `fdsFooButton`, etc)

Reply:
Upcoming button components will replace the shim, so I'm not worried about collisions here